mava/utils/jax.py

```diff
@@ -59,3 +59,9 @@ def unreplicate_learner_state(
     and one for the `update batch size`.
     """
     return jax.tree_map(lambda x: x[(0,) * unreplicate_depth], learner_state)  # type: ignore
+
+
+def unreplicate_batch_dim(x: Any) -> Any:
+    """Unreplicated just the batch dim.
```

Comment:
maybe specify that it is the update_batch_dim and getting the element 0 is fine as the params are replicated (broadcasted) over that batch